src/stdfblib/events/GEN_E_DEMUX_fbt.cpp

```diff
@@ -64,17 +59,17 @@ bool GEN_E_DEMUX::createInterfaceSpec(const char *paConfigString, SFBInterfaceSp
       paInterfaceSpec.mNumEOs = static_cast<TEventID>(forte::core::util::strtoul(acPos, nullptr, 10));
 
       if(paInterfaceSpec.mNumEOs < CFunctionBlock::scmMaxInterfaceEvents){
-        mEventOutputNames = new CStringDictionary::TStringId[paInterfaceSpec.mNumEOs];
-        mEventOutputTypeIds = new CStringDictionary::TStringId[paInterfaceSpec.mNumEOs];
+        mEventOutputNames.reset(new CStringDictionary::TStringId[paInterfaceSpec.mNumEOs]);
+        mEventOutputTypeIds.reset(new CStringDictionary::TStringId[paInterfaceSpec.mNumEOs]);
         const CStringDictionary::TStringId eventID = CStringDictionary::getInstance().getId("Event");
-        std::memset(mEventOutputTypeIds, static_cast<int>(eventID), sizeof(*mEventOutputTypeIds) * paInterfaceSpec.mNumEOs);
-        generateGenericInterfacePointNameArray("EO", mEventOutputNames, paInterfaceSpec.mNumEOs);
+        std::fill_n(mEventOutputTypeIds.get(), paInterfaceSpec.mNumEOs, eventID);
```

Comment:
I believe this array is entirely unnecessary due to https://github.com/eclipse-4diac/4diac-forte/pull/273#discussion_r1839765837.